sdk/synapse/azartifacts/README.md

```diff
@@ -0,0 +1,74 @@
+# Azure Synapse Artifacts Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/synapse/azartifacts)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/synapse/azartifacts)
+
+The `azartifacts` module provides operations for working with Azure synapse.
+
+[Source code](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/synapse/azartifacts)
```

Comment:
The CI is not working therefore this link is not reported as invalid. Therefore the CI pipelines are designed not to work in the first PR of a module?

Reply:
We still need to figure out how this will work long-term.  At present, we can assume that CI won't be hooked up for the first module release :(

Reply:
That would be the most unfortunate... do we have to locally run `go vet` to verify the SDK could compile properly?

Reply:
That's what I did yes.  Long term, we'll want to have an easy way to set up CI before releasing a module.  I didn't bother with it here given our requirement for expedience.